service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusServiceIT.java

```diff
@@ -54,14 +60,44 @@ public class EhrExtractStatusServiceIT {
     @Autowired
     private EhrExtractStatusRepository ehrExtractStatusRepository;
 
+    @Autowired
+    private EhrExtractTimeoutScheduler ehrExtractTimeoutScheduler;
+
     @MockBean
     private TimestampService timestampService;
 
+    @MockBean
+    private Logger logger;
+
     @BeforeEach
     public void emptyDatabase() {
         ehrExtractStatusRepository.deleteAll();
     }
 
+    @Test
+    void updateEhrExtractStatusAckTest() {
+        var inProgressConversationId = generateRandomUppercaseUUID();
+
+        EhrExtractStatusService ehrExtractStatusServiceSpy = spy(ehrExtractStatusService);
```

Comment:
In terms of consistency - if using `var` let's keep this:

```suggestion
        var ehrExtractStatusServiceSpy = spy(ehrExtractStatusService);
```